test/unit/cache_store_test.rb

```diff
@@ -20,7 +20,8 @@ class CacheStoreTest < ActionView::TestCase
     # test:
     #   - :memory_store
     #   - :size: 5000
-    options = { size: 5000 }
+    #   - :compress: true
+    options = { size: 5000, compress: true }
```

Comment:
The default value is now set for `:compress`

![Screenshot from 2024-03-12 12-20-53](https://github.com/3scale/porta/assets/1270328/2a0416d2-ad14-4c37-918c-49d0f5033119)

Reply:
Do we need to compress?

Reply:
Damn, I don't understand this change anymore :sweat_smile:  I mean, judging by the diff in Rails, the default value should be `false`, why it's set to `true`, but the test passes? :upside_down_face: 

I'll need to check.

Also, I don't know whether we need it or not. Thoughts?

Reply:
So, we don't actually enable it, it's just a test. The test started failing because the expected value was `{ size: 5000 }`, and the actual value was `{ size: 5000, compress: false }` , because this is what Rails adds.

So, I changed the test to get the same expected result.